Mail/Views/Thread/MessageBodyView.swift

```diff
@@ -85,14 +75,31 @@ struct MessageBodyView: View {
 
     private func loadBody(blockRemoteContent: Bool) {
         Task {
+            var messageBody = model.showBlockQuote ? presentableBody.body?.value : presentableBody.compactBody
+
+            if presentableBody.body?.type == "text/plain" {
+                messageBody = createHTMLForPlainText(text: messageBody)
+            }
+
             let loadResult = await model.loadHTMLString(
-                value: model.showBlockQuote ? presentableBody.body?.value : presentableBody.compactBody,
+                value: messageBody,
                 blockRemoteContent: blockRemoteContent
             )
 
             displayContentBlockedActionView = (loadResult == .remoteContentBlocked)
         }
     }
+
+    private func createHTMLForPlainText(text: String?) -> String {
```

Comment:
Couldn't we do the processing from the webview model?
I don't have a strong opinion, what do you think?